packages/odyssey-storybook/src/guidelines/DocsStatus.stories.mdx

```diff
@@ -0,0 +1,233 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Guidelines/Docs Status" />
+
+# Docs Status
+
+This document serves as a scratch pad for the docs migration effort.
```

Comment:
I love this. I would feel totally comfortable with merging this PR and providing transparency as to what remains in this way.